cmd/datamon/cmd/flags.go

```diff
@@ -472,3 +472,7 @@ func requireFlags(cmd *cobra.Command, flags ...string) {
 		}
 	}
 }
+
+func dumpContext(flags flagsT) string {
```

Comment:
suggestion: either

* scope `dumpContext` locally in `label_set.go`'s `Run` `func`

or

* add `dumpContext` as a method to `cliOptionInputs`

this suggestion is based on, most recently, [commentary](https://github.com/oneconcern/datamon/pull/379#discussion_r373773921) on #379 , specifically

> the new `cliOptionInput` (without the cli prefix...) should bear more responsibilities, and be shed from `cmd`

Reply:
ok